doc/models/about_models.rst

```diff
@@ -20,3 +20,76 @@ We also define a few general Pretrainers in a submodule here. The Pretrainers in
 this top-level submodule are suitable to pre-train a large class of models
 (e.g., any model that encodes sentences), while more task-specific Pretrainers
 are found in that task's submodule.
+
+Below, we describe a few popular models that we've implemented and include our
+output when training.
+
+Attention Sum Reader
+--------------------
+
+The `Attention Sum Reader
+<https://www.semanticscholar.org/paper/Text-Understanding-with-the-Attention-Sum-Reader-Kadlec-Schmid/1023b20d226bd0af9fdf0fd1847accefbfa5ec84>`_
+Network is implemented in
+:mod:`~deep_qa.models.reading_comprehension.attention_sum_reader`.
+
+Here are the train logs::
+
```

Comment:
Ok, don't bother spending time in your last two days worrying about this.  But can you add an issue to look into whether we can do this?